tests/test_class_conversion.rs

```diff
@@ -122,9 +122,9 @@ fn test_polymorphic_container_does_not_accept_other_types() {
 
     let setattr = |value: PyObject| p.as_ref(py).setattr("inner", value);
 
-    assert!(setattr(1i32.into_py(py)).is_err());
+    assert!(setattr(1i32.into_object(py)).is_err());
```

Comment:
Users migrating might have to replace `into_py` with `into_object` where they to want `PyObject` output. I'm not sure how annoying this might be.

Worst case I think we instruct users to replace all `.into_py()` with `.into_object()`, and then they can case-by-case change back to `into_py()` where the better typing is helpful.